### PR TITLE
New Checklist for PR Template

### PR DESCRIPTION
**Pull request by goplayoutside3**
As discussed in frontend standup, this is for revising FEM's PR review checklist. To easily read the new checklist, open the changed file outside of this PR either in Github or by pulling locally. The diff here can be confusing.

Please keep comments and discussion scoped by referencing lines of code.

### CODE REVIEW

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,49 +1,54 @@
...
 - What Zooniverse project should my reviewer use to review UX?
 - What user actions should my reviewer step through to review this PR?
 - Which storybook stories should be reviewed?
 - Are there plans for follow up PR’s to further fix this bug or develop this feature?
 
-
-# Review Checklist
+# Checklist
+_PR Creator - Please cater the checklist to fit the review needed for your code changes._
+_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._
```
Comment (goplayoutside3) [inline docs/PULL_REQUEST_TEMPLATE.md:18]: My goal here is to encourage the PR creator to step through the relevant checklist items themself _before_ pushing a new branch and requesting review. I think syncing the PR creator and PR reviewer's understanding of the PR's changes will be beneficial to our team's limited time resources.

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,49 +1,54 @@
...
+- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari.
+- [ ] FEM works in a mobile browser.
+
+## General UX
+- [ ] All pages of a FEM project load: Home Page, Classify Page, and About Pages
+- [ ] Can submit a classification
+- [ ] Can sign-in and sign-out
+- Example Staging Projects
+    - [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
+    - [Transformers](https://local.zooniverse.org:3000/projects/darkeshard/test-project-2022)
```
Comment (goplayoutside3) [inline docs/PULL_REQUEST_TEMPLATE.md:33]: Should it be required to load and make a demo classification to TESS in this category?

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,49 +1,54 @@
...
-- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
-  - [ ] Is this approved by our designer?
-- [ ] Is this ready for production deployment?
+- [ ] Tests are passing locally and on Github.
+- [ ] Documentation is up to date. Changelog has been updated if appropriate.
+- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected.
+- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari.
+- [ ] FEM works in a mobile browser.
+
+## General UX
```
Comment (eatyourgreens) [inline docs/PULL_REQUEST_TEMPLATE.md:27]: Accessibility testing should be included here, or should be in its own top level Accessibility section immediately after General UX.

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,49 +1,54 @@
...
+- [ ] Unit tests are added or updated.
+
+## New Feature
+- [ ] The PR creator has listed user actions to use when testing the new feature.
+- [ ] Unit tests are included for the new feature.
+- [ ] A storybook story been created or updated.
+- [ ] The component is accessible.
+  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
+  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
+  - It is passing accessibility checks in the storybook.
```
Comment (eatyourgreens) [inline docs/PULL_REQUEST_TEMPLATE.md:47]: Accessibility should also be tested on bug fixes and refactors, to make sure that regressions haven't been introduced.

**Review by eatyourgreens — CHANGES_REQUESTED** [review]

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -1,49 +1,54 @@
...
-- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
-- [ ] Is the new feature working or bug now fixed?
-  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
-- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
-  - [ ] Is this approved by our designer?
-- [ ] Is this ready for production deployment?
+- [ ] Tests are passing locally and on Github.
+- [ ] Documentation is up to date. Changelog has been updated if appropriate.
+- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected.
+- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari.
```
Comment (eatyourgreens) [inline docs/PULL_REQUEST_TEMPLATE.md:24]: Should we note that we have a Browserstack account for cross-browser testing?

File: docs/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (eatyourgreens) [reply docs/PULL_REQUEST_TEMPLATE.md:33]: Given that the new platform is mostly used for transcription projects at the moment, they should probably feature here.

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -1,49 +1,54 @@
...
+
+## Bug Fix
+- [ ] The PR creator has listed user actions to use when testing if bug is fixed.
+- [ ] The bug is fixed.
+- [ ] Unit tests are added or updated.
+
+## New Feature
+- [ ] The PR creator has listed user actions to use when testing the new feature.
+- [ ] Unit tests are included for the new feature.
+- [ ] A storybook story been created or updated.
```
Comment (eatyourgreens) [inline docs/PULL_REQUEST_TEMPLATE.md:43]: Should we also encourage documenting the new component on its Storybook Docs page?

Comment (eatyourgreens) [reply docs/PULL_REQUEST_TEMPLATE.md:43]:
```suggestion
- [ ] A storybook story has been created or updated.
```
Small typo here.

**Comment by goplayoutside3** [comment]
An example of what the template looks like rendered as markdown:

_Please request review from `@zooniverse/frontend` team or an individual member of that team._ 

## Package
app-project

## Linked Issue and/or Talk Post
1234

## Describe your changes
Here's where I talk about the code changes in this PR

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What Zooniverse project should my reviewer use to review UX?
- What user actions should my reviewer step through to review this PR?
- Which storybook stories should be reviewed?
- Are there plans for follow up PR’s to further fix this bug or develop this feature?

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [ ] Tests are passing locally and on Github
- [ ] Documentation is up to date anc dhangelog has been updated if appropriate
- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
- [ ] All pages of a FEM project load: Home Page, Classify Page, and About Pages
- [ ] Can submit a classification
- [ ] Can sign-in and sign-out
- [ ] The component is accessible
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook


## Bug Fix
- [ ] The PR creator has listed user actions to use when testing if bug is fixed
- [ ] The bug is fixed
- [ ] Unit tests are added or updated

## New Feature
- [ ] The PR creator has listed user actions to use when testing the new feature
- [ ] Unit tests are included for the new feature
- [ ] A storybook story has been created or updated
  - Example of SlideTutorial [component](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/SlideTutorial/SlideTutorial.js) with docgen comments, and its [story](https://zooniverse.github.io/front-end-monorepo/@zooniverse/classifier/index.html?path=/docs/other-slidetutorial--default)

## Refactoring
- [ ] The PR creator has described the reason for refactoring
- [ ] The refactored component(s) continue to work as expected

## Maintenance
- [ ] If not from dependabot, the PR creator has described the update (major, minor, or patch version, changelog)

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -1,49 +1,54 @@
...
+- [ ] The component is accessible
+  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
+  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
+  - It is passing accessibility checks in the storybook
+
+
+## Bug Fix
+- [ ] The PR creator has listed user actions to use when testing if bug is fixed
+- [ ] The bug is fixed
+- [ ] Unit tests are added or updated
```
Comment (eatyourgreens) [inline docs/PULL_REQUEST_TEMPLATE.md:41]: I’m not sure of the exact wording I’d use for this but: is there a test that catches the bug?

This isn’t true for all bugs, but a fix that starts with a failing test, followed by changes that pass the test, is easier to review.

**Review by eatyourgreens — APPROVED** [review]

File: docs/PULL_REQUEST_TEMPLATE.md (hunk 7 above)
Comment (goplayoutside3) [reply docs/PULL_REQUEST_TEMPLATE.md:41]: Good idea! I think the refactoring of unit tests changes a lot on a case by case basis, and TDD as a concept is a better fit for our frontend team playbook. I'll leave the wording as it for now, and we can re-edit in the future.

**Pull request merged**